integration/shareddata/scalars.go

```diff
@@ -28,7 +28,10 @@ const (
 	doubleBig     = float64(1 << 61)   // 2305843009213693952.0: some number larger than safe integer (doubleBig+1 == doubleBig)
 	longBig       = int64(1 << 61)     // 2305843009213693952:   same as doubleBig but integer
 
-	// TODO https://github.com/FerretDB/FerretDB/issues/2321
+	doubleMaxOverflow = float64(9223372036854776833)  // double value that is larger than MaxInt64
+	doubleMinOverflow = float64(-9223372036854776833) // double value that is smaller than MinInt64
```

Comment:
So could it be `float64(math.MinInt64) - 1.0)`, with a value in a comment?

Overall, we have a strange mix of expressions and literals.

Reply:
subtrackting 1.0 will probably result in the same number because of the lack of precision

Reply:
-2.0 then? After all, we have a test for that in init